src/main/java/de/helmholtz/marketplace/cerebrum/entities/Image.java

```diff
@@ -0,0 +1,30 @@
+package de.helmholtz.marketplace.cerebrum.entities;
+
+import org.neo4j.ogm.annotation.GeneratedValue;
+import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.NodeEntity;
+
+@NodeEntity
+public class Image {
```

Comment:
Because the source of the image could be an url or base64 encoded. In the API draft the image is an object. I can change it to a String, but then you have to find out whether it is a url or base64 by your own.

Reply:
Hmm... this is a tricky one. I will suggest we stick to pure string here for now